crm_request/models/request.py

```diff
@@ -101,3 +101,15 @@ def message_post(self, **kwargs):
                     'crm_request', 'stage_wait_customer')[1]
 
         return result
+
+    @api.multi
+    def write(self, values):
+        # Get all followers before change
+        super(CrmClaim, self)._get_followers
+
+        #  unsubscribe all followers before change
+        super(CrmClaim, self).message_unsubscribe(
+            partner_ids=[self.message_partner_ids.id])
+
+        # launching the standard process "write"
+        return super(CrmClaim, self).write(values)
```

Comment:
You should start by the "super", thus you avoid to maybe add again the current user as follower.